gspread/worksheet.py

```diff
@@ -3201,3 +3202,51 @@ def cut_range(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def add_validation(
+        self,
+        source: str,
+        condition_type: ValidationConditionType,
+        *values: Iterable[Any],
+        **kwargs,
```

Comment:
Please remove the `kwargs` and use explicit arguments: `inputMessage`, `strict`, `showCustomUi`.

We tend to remove most kwargs and prevent using them when not necessary.

Reply:
Added in 06c3f24113244b1b8a5f285e5b5976db396d3a22